src/output/timely_opt.rs

```diff
@@ -47,8 +47,13 @@ pub(super) fn compute(dump_enabled: bool, mut all_facts: AllFacts) -> Output {
     tx.send(all_facts).unwrap();
     mem::drop(tx);
     let rx = Mutex::new(rx);
+    let mut dataflow_arg = Vec::new();
+    if workers > 1 {
+        let item = format!("{}{}", "w", workers);
+        dataflow_arg.push(item);
```

Comment:
same here; maybe we want a helper fn :)

Reply:
for helper function, any ideal place where I should store this? or should I create common helper functions file for that?

Reply:
@nikomatsakis done!